_data/projects.yml

```diff
@@ -226,17 +213,12 @@ of software projects, including the
       image: "img/projects/cachecash_diagram.png"
       site:
       status: *stealth
-      description: "Delivery of large chunks of content, such as video, accounts for a
-substantial percentage of all Internet traffic.  This content is usually
-served by provider networks that contract with Internet Service Providers.
-CacheCash provides a similar service in which interested
-users run caches, and are incentivized to participate by receiving
-a crypto-currency (Cachecoin) in exchange for serving content to other users.
-Both cryptographic and economic techniques demonstrate that CacheCash users are either forced or
-economically incentivized to honestly serve content. Our analysis
-shows that CacheCash scales to meet the workload of even the most popular
-services used today. By building CacheCash, we intend to change CDNs by
-more readily and pervasively including end-user served content."
+      description: "The emergence of distributed systems to process service requests for content, and the use of 
```

Comment:
I wouldn't change this text.  Let's just leave this for now.  I'll rewrite it (possibly w/ Ghada) at some point soon.

Reply:
Ok. I replaced the old text